### ISSUE TITLE
One feed per country

### ISSUE THREAD

**Issue by skinkie** [public-transport/european-transport-feeds#22]
@juliuste you mention in in the pull requests that you would prefer one feed per country. If that feed is provided as separate files, how would you link to them? Creating a zip files, just to have a single file, would defeat the purpose.

**Comment by eutampieri** [comment]
Same problem for #20 : every region has its set of NeTeX feeds, one for each level

**Comment by juliuste** [comment]
Fair question. In my opinion, the effort for merging different feeds is a big problem worthy of its own repository (at least from my experience in Germany, where local feed URLs change all the time and you have to keep on top of that). I therefore think that it probably makes sense to keep these projects out of this repo, for the time being. I'd be very happy to include such a community-generated feed on this list then, though.

That being said, I'm also interested in generating joined feeds, even if that happens in a separate repository. Just to make sure: @vkrause @felixguendling would it be possible to do that as part of the [transitous](https://github.com/public-transport/transitous) project, since you already are collecting feeds there anyway? I know its not your primary use-case, but having multiple places in which people collect and merge feeds probably wouldn't make too much sense either…

**Comment by felixguendling** [comment]
> ... having multiple places in which people collect and merge feeds 

I think **collecting feeds** (projects like Transitland, Mobility Database, Transitous) and **merging feeds** (what OTP and MOTIS already do internally, using OSM to find transfers between locations of different feeds) are two completely different things and I don't see value in mixing those two tasks in one repository.

In terms of being able to do routing, show departure/arrival boards, compute isochrones, etc. MOTIS does the job already for multiple feeds (I lost track of how many there are now in Transitous). If you are interested in something else you can either implement it in MOTIS as well or start from scratch by writing a "gtfs-merge" tool (maybe on top of gtfs-clean?).

**Comment by vkrause** [comment]
There might be something inbetween collecting feeds (Transitous) and merging feeds (MOTIS) though that we also need here: Instructions on how to merge feeds. We had cases where one feed contained strictly better information for certain parts than another one, or feeds that contain split-up cross-border routes that might need to be stitched together again, ie. things that go beyond what a generic merger might be able to resolve without external guidance.

**Comment by felixguendling** [comment]
That's true. For Transitous, I would see the tracking of this information (about "how to merge feeds") on the Transitous side (where also the Lua scripts and the datasets themself are tracked), not on the MOTIS side. But MOTIS probably should gain functionality to use these instructions on how to merge feeds (e.g. in form of more [Lua scripting](https://mastodon.social/@felixguendling/115024110168538521)). At least I see this as a more realistic option than to process a European-wide GTFS feed with gtfs-clean or SQL database (Postgres/DuckDB) tooling.
